README.md

```diff
@@ -1600,10 +1608,9 @@ Long-term capital gains tax is often lower than ordinary income tax.
 Many investors hold assets for longer than a year in order to qualify for the lesser tax
 burden of long-term capital gains.
 
-🄳 **Alternative minimum tax (AMT)** is an
-[entirely separate kind of tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that is
-neither ordinary income tax nor employment tax, and has its own rules and only applies in
-some situations.
+🄳 **Alternative minimum tax (AMT)** is neither ordinary income tax nor employment tax, but
```

Comment:
Let's wrap this and the next paragraph together:

🄳 **Alternative minimum tax (AMT)** is a
[supplemental income tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that applies to certain individuals and in some situations. This type of tax does not come up for many taxpayers, but higher income earners and people in special situations may have to pay large AMT bills. AMT was first enacted in 1982 is not the same as ordinary income tax or employment tax.

Reply:
1979*

Reply:
Revised. Improvement, expansion still needed.